CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.14 - February 24, 2021
+
+## Breaking changes
+* [map] Change signature of Map#getElevation from Float to Double [#120](https://github.com/mapbox/mapbox-maps-android/pull/120)
+* [map] Fixed strings are now returned as formatted expressions [#120](https://github.com/mapbox/mapbox-maps-android/pull/120)
```

Comment:
There should also be a breaking change that `data` field can not be empty for `GeoJsonSource`. or actually we can call it a known bug.